source/_integrations/octoprint.markdown

```diff
@@ -22,8 +22,8 @@ ha_platforms:
 {% include integrations/config_flow.md %}
 
 ### API Key
-
-The Octoprint integration will attempt to register itself via the [application keys plugin](https://docs.octoprint.org/en/master/bundledplugins/appkeys.html). After submitting the configuration UI in Home Assistant, open the Octoprint UI and click allow on the prompt.
+For the integration to work, please check that the plugin Discovery is enabled (you can disable it after integration is done).
```

Comment:
That comment is incorrect now, as discovery should be enabled even afterward.

Reply:
Sorry, I've missed this information.